recipes/rdkit/2016.03.1/meta.yaml

```diff
@@ -22,7 +23,7 @@ requirements:
     - requests
     - boost >=1.48.0,{{CONDA_BOOST}}*
     - python
-    - numpy
+    - numpy >=1.7
```

Comment:
Right, will add this.

On Mon, Aug 1, 2016 at 9:17 AM, Björn Grüning <notifications@github.com>
wrote:

> In recipes/rdkit/2016.03.1/meta.yaml
> <https://github.com/bioconda/bioconda-recipes/pull/2072#discussion_r72932498>
> :
>
> > @@ -22,7 +23,7 @@ requirements:
> >      - requests
> >      - boost >=1.48.0,{{CONDA_BOOST}}*
> >      - python
> > -    - numpy
> > +    - numpy >=1.7
>
> For runtime dependency as well?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bioconda/bioconda-recipes/pull/2072/files/f87f2080acdb44d74abdcfaf070953e9fadcb32a..2412136a3be50247841501e364de33749161e885#r72932498>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAG6sR0v-9tSVFg_RCX0o7I2giWDnnSpks5qbZ17gaJpZM4JZTfh>
> .
>



-- 
--------------------------------------------------
Daniel Maticzka, Dipl. Inf.
Lehrstuhl für Bioinformatik
Institut für Informatik
Albert-Ludwigs-Universität Freiburg

Tel: (+49) 761 203 97529
Fax: (+49) 761 203 7462
Mail: maticzkd@informatik.uni-freiburg.de
--------------------------------------------------